accounts/accounts.go

```diff
@@ -5,11 +5,12 @@ import (
 )
 
 const (
-	accountsPath        = "accounts"
-	entitiesPath        = "entities"
-	instrumentsPath     = "instruments"
-	payoutSchedulesPath = "payout-schedules"
-	filesPath           = "files"
+	accountsPath           = "accounts"
+	entitiesPath           = "entities"
+	instrumentsPath        = "instruments"
+	payoutSchedulesPath    = "payout-schedules"
+	filesPath              = "files"
+	PaymentInstrumentsPath = "payment-instruments"
```

Comment:
There is no need to export this const. Rename it to `paymentInstrumentsPath`

Reply:
👍